tests/util_tests.py

```diff
@@ -79,3 +80,22 @@ def runTest(self):
         self.assertTrue(isinstance(self.abstracts_preprocessor.get_term_frequency_sparse_matrix(), sparse.csr_matrix))
         self.assertEqual(max(map(lambda inp: len(inp.split(' ')), self.abstracts_preprocessor.abstracts.values())),
                          self.abstracts_preprocessor.get_num_units())
+
+
+class TestModelInitializer(unittest.TestCase):
+    def runTest(self):
+        users_cnt = 10
+        base_dir = os.path.dirname(os.path.realpath(__file__))
+        with open(os.path.join(os.path.dirname(base_dir), 'config/recommender.json')) as data_file:
+            json_config = json.load(data_file)
+        config = RecommenderConfiguration().get_hyperparameters()
+        initializer = ModelInitializer(config, 1)
+        path = initializer._create_path('user_v', users_cnt)
+        self.assertTrue(path.endswith('n_iterations:1,n_rows:10user_v.dat'))
```

Comment:
Too many characters that I don't prefer in the file name, I prefer an encoding function if you need to list all these in the file name.